plugins/statistics.koplugin/main.lua

```diff
@@ -2643,6 +2731,15 @@ end
 
 
 function ReaderStatistics:onPosUpdate(pos, pageno)
+    if self.use_pagemap_for_stats then
+        local page_sequence_number = self.document:getPageMapCurrentPageLabel()
```

Comment:
Label or index?

Reply:
Should be the 2nd returned value, the sequential index (among the 3rd-item-total-refpage-count).